build/rpms.in.yaml

```
@@ -0,0 +1,19 @@
+contentOrigin:
+  repos:
+    - repoid: ubi-9-for-${basearch}-baseos-rpms
+      baseurl: https://cdn-ubi.redhat.com/content/public/ubi/dist/ubi9/9/${basearch}/baseos/os/
+    - repoid: ubi-9-for-${basearch}-appstream-rpms
+      baseurl: https://cdn-ubi.redhat.com/content/public/ubi/dist/ubi9/9/${basearch}/appstream/os/
+packages:
+  - jq
+  - tar
+  - gzip
+  - rsync
+  - findutils
+context:
+  containerfile: Dockerfile.rhtap
+arches:
+  - aarch64
+  - x86_64
+  - s390x
+  - ppc64le
```

(nit) Missing trailing newline
```suggestion
  - ppc64le

```